Dockerfile

```diff
@@ -6,10 +6,41 @@ RUN apt-get update -y && \
 
 RUN LC_ALL=en_US.UTF-8 add-apt-repository ppa:ondrej/php && \
   apt-get update -y && \
-  apt-get install -y git-core unzip npm nodejs-legacy wget fontconfig php7.0 php7.0-xml php7.0-mbstring php7.0-curl php7.0-gd php7.0-zip php7.0-intl php7.0-sqlite3 php7.0-mysql php7.0-pgsql php7.0-soap php7.0-phpdbg && \
+  apt-get install -y git-core unzip npm nodejs-legacy wget fontconfig php7.0 php7.0-xml php7.0-mbstring php7.0-curl php7.0-gd php7.0-zip php7.0-intl php7.0-sqlite3 php7.0-mysql php7.0-pgsql php7.0-soap php7.0-phpdbg php-pear php-xml php7.0-dev && \
   apt-get install -y php-redis php-memcached php-imagick && \
   apt-get upgrade -y && \
   rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
+#ORACLE INSTALLATION
+#get the zips
+RUN mkdir /opt/oracle && \
+  wget -O /opt/oracle/instantclient-basic-linux.x64-12.1.0.2.0.zip \
+  https://github.com/DeepDiver1975/oracle_instant_client_for_ubuntu_64bit\
+/raw/12.1.as.zip/zips/instantclient-basic-linux.x64-12.1.0.2.0.zip && \
+  wget -O /opt/oracle/instantclient-sdk-linux.x64-12.1.0.2.0.zip \
+  https://github.com/DeepDiver1975/oracle_instant_client_for_ubuntu_64bit\
+/raw/12.1.as.zip/zips/instantclient-sdk-linux.x64-12.1.0.2.0.zip && \
+  wget -O /opt/oracle/instantclient-sqlplus-linux.x64-12.1.0.2.0.zip \
+  https://github.com/DeepDiver1975/oracle_instant_client_for_ubuntu_64bit\
+/raw/12.1.as.zip/zips/instantclient-sqlplus-linux.x64-12.1.0.2.0.zip
+
+#unzip them
+RUN unzip /opt/oracle/instantclient-basic-linux.x64-12.1.0.2.0.zip -d /usr/lib/oracle && \
```

Comment:
I think it's cleaner to keep the files in opt oracle and reference that folder in the pecl command below ...